configs/controlnet_animation/anythingv3_config.py

```diff
@@ -0,0 +1,36 @@
+# config for model
+stable_diffusion_v15_url = 'Linaqruf/anything-v3.0'
+controlnet_hed_url = 'lllyasviel/sd-controlnet-hed'
+control_detector = 'lllyasviel/ControlNet'
+control_scheduler = 'UniPCMultistepScheduler'
```

Comment:
This variable seems not to be used. 👀

Reply:
use this variable to change inference scheduler with 'from_config'